.github/workflows/ci.yml

```diff
@@ -20,4 +20,10 @@ jobs:
         run: ./gradlew assemble
       - name: Test
         run: ./gradlew test && ./gradlew -p "android-test" test
-        
+      - name: Store test results
+        if: success() || failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: test-results
+          path: "build/test-results/test/TEST-*.xml"
```

Comment:
@LikeTheSalad , can you please help me to know where the JUnit test results can be found? I somehow thought it will be there, but it's not  :/

Reply:
Hi @v1v - looks good! although the `build` dir can be within other modules, so I think this should work: `**/build/test-results/{test,testRelease*}/TEST-*.xml` - Also, for Android projects, there's 2 build variants (debug and release), and both are tested using the same tests, so it's kind of a duplication of tests per variant, which is why I picked the `release` tests results only in the path I shared above. Feel free to modify it as you need :)

Reply:
```suggestion
          path: "**/build/test-results/{test,testRelease*}/TEST-*.xml"
```

Reply:
Somehow its not found it yet

<img width="1267" alt="image" src="https://user-images.githubusercontent.com/2871786/200298277-d9778d1f-1b8c-467c-8e47-af45fb81e5f6.png">